src/main/resources/application.properties

```diff
@@ -12,9 +12,4 @@ cirq.directory=cirq
 cirq.directory.max-cut=${cirq.directory}/max-cut
 
 examples.directory=examples
-examples.directory.max-cut=${examples.directory}/max-cut
```

Comment:
We're not using them currently, but in the future we'd still like to have an automatic way to load example problems that doesn't have problems when being built into a .jar. Should we really remove this if we want to make use of it again in the future?